DependencyInjection/HttplugExtension.php

```diff
@@ -81,6 +81,10 @@ public function load(array $configs, ContainerBuilder $container)
         $this->configureClients($container, $config);
         $this->configurePlugins($container, $config['plugins']); // must be after clients, as clients.X.plugins might use plugins as templates that will be removed
         $this->configureAutoDiscoveryClients($container, $config);
+
+        if (!class_exists(\Twig_Environment::class)) {
+            $container->removeDefinition('httplug.collector.twig.http_message');
```

Comment:
this should be inside the `if ($profilingEnabled) {` i think, as the service is defined in data-collector.xml.

Reply:
Good one, changed.